libraries/chain/account_evaluator.cpp

```diff
@@ -280,12 +273,6 @@ void_result account_update_evaluator::do_evaluate( const account_update_operatio
       FC_ASSERT( !o.extensions.value.owner_special_authority.valid() );
       FC_ASSERT( !o.extensions.value.active_special_authority.valid() );
```

Comment:
Please add a comment why HARDFORK_516 has to stay, to simplify future cleanups.

Reply:
Ran a snapshot for mainnet, and grep'd logs with no trace of 516 changing anything. Removing.